server/src/app.ts

```diff
@@ -57,36 +57,23 @@ async function startServer() {
 
     app.get("/api/v1/recipes", async (req: Request, res: Response) => {
       try {
-        const collection: Collection = db.collection("recipes");
-        const documents = await collection.find().toArray();
-
-        const recipes: Recipe[] = documents.map((recipe) => ({
-          id: recipe._id,
-          title: recipe.title,
-          description: recipe.description,
-          categories: recipe.categories,
-          ingredients: recipe.ingredients,
-          steps: recipe.steps,
-          createdAt: recipe.createdAt,
-        }));
-
-        res.status(200).json(recipes);
+        const allRecipes = await recipesRepository.getAllRecipes();
+
+        res.status(200).json(allRecipes);
       } catch (error) {
         console.error("Error getting recipes:", error);
         res.status(500).json({ error: "Failed to get recipes" });
       }
     });
 
     app.get("/api/v1/recipes/:id", async (req: Request, res: Response) => {
-      if (!ObjectId.isValid(req.params.id)) {
+      const recipeId = req.params.id;
+      if (!ObjectId.isValid(recipeId)) {
         res.status(400).json({ error: "Not valid document ID" });
```

Comment:
if (!ObjectId.isValid(recipeId)) {
       **return** res.status(400).json({ error: "Not valid document ID" });
}